packages/protocol/test/governance/network/governance.ts

```diff
@@ -1732,164 +1709,275 @@ contract('Governance', (accounts: string[]) => {
           // @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
           { value: minDeposit }
         )
-        await timeTravel(newDequeueFrequency, web3)
-        await governance.approve(proposalId1, index1)
-        await governance.propose(
-          [transactionSuccess2.value],
-          [transactionSuccess2.destination],
-          // @ts-ignore bytes type
-          transactionSuccess2.data,
-          [transactionSuccess2.data.length],
-          descriptionUrl,
-          // @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
-          { value: minDeposit }
-        )
-        await timeTravel(newDequeueFrequency, web3)
-        await governance.approve(proposalId2, index2)
-        await timeTravel(approvalStageDuration, web3)
-        await mockLockedGold.setAccountTotalLockedGold(account, weight)
-      })
-
-      it('should set mostRecentReferendumProposal to the youngest proposal voted on', async () => {
-        await governance.vote(proposalId2, index2, value)
-        await governance.vote(proposalId1, index1, value)
-        const mostRecent = await governance.getMostRecentReferendumProposal(accounts[0])
-        assert.equal(mostRecent.toNumber(), proposalId2)
-      })
-
-      it('should return true on `isVoting`', async () => {
-        await governance.vote(proposalId2, index2, value)
-        await governance.vote(proposalId1, index1, value)
-        const voting = await governance.isVoting(accounts[0])
-        assert.isTrue(voting)
+        await timeTravel(dequeueFrequency, web3)
+        const otherProposalId = 2
+        await assertRevert(governance.vote(otherProposalId, index, value))
       })
 
-      describe('after the first proposal expires', () => {
+      describe('when voting on two proposals', () => {
+        const proposalId1 = 2
+        const proposalId2 = 3
+        const index1 = 1
+        const index2 = 2
         beforeEach(async () => {
+          const newDequeueFrequency = 60
+          await governance.setDequeueFrequency(newDequeueFrequency)
```

Comment:
I would say that only changes that I made are regarding removal/update of some await timeTravel and padding since I encapsulated part of existing tests into describe('when proposal is approved'). I believe that padding is what is confusing github